core/src/app/halma/AboutScreen.java

```diff
@@ -65,6 +73,12 @@ public void clicked(Actor a) {
             link = "https://github.com/mmelnizky";
         if(a == linkToMarvin)
             link = "https://github.com/Crazy-Marvin";
+        if (a == helpWIthTranslation)
+            link = "https://hosted.weblate.org/engage/halma/";
+        if (a == termsOfService)
```

Comment:
I thought about Terms of Service and came to the conclusion that they are not needed. Please remove them.
Sorry for the confusion.